CHANGELOG.asciidoc

```diff
@@ -110,7 +112,6 @@ This release also includes changes from <<release-3-2-6, 3.2.6>>.
 * Removed access to previously deprecated `CoreGremlinPlugin#INSTANCE` field.
 * `gremlin.sh` and `gremln.bat` no longer support the option to pass a script as an argument for execution mode without using the `-i` option.
 * Graphite and Ganglia are no longer packaged with the Gremlin Server distribution.
-* `TransactionException` is no longer a class of `AbstractTransaction` and it extends `RuntimeException`.
```

Comment:
Why does this get removed from an earlier version?

Reply:
That was unintentional, not show how/why/when that happened. I will add it back.